idx/backward-compatibility/migrate-old-table.php

```diff
@@ -29,8 +52,17 @@ public function grab_post_ids() {
 		);
 	}
 
+	/**
+	 * Update Post Type.
+	 *
+	 * @access public
+	 * @param mixed $post_id Post ID.
+	 * @param mixed $link Link.
+	 * @param mixed $post_type Post Type.
+	 * @return void
+	 */
 	public function update_post_type( $post_id, $link, $post_type ) {
-		if ( $post_type === 'idx_page' ) {
+		if ( 'idx_page' === $post_type ) {
```

Comment:
Changed to Yoda comparison you did

Reply:
Yoda conditions you must - https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#yoda-conditions